server/@types/bapv.d.ts

```diff
@@ -11,6 +11,7 @@ export type Booker = {
 export type AddPrisonerJourney = {
   supportedPrisonIds: string[]
   selectedPrisonId?: string
+  prisonerDetails?: Record<string, string>
```

Comment:
This is to save the prisoner details a user has entered in case their request fails and they want to go back again and re-try after amending.